app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/previous_return_exists_with_same_completion_date.scala.html

```diff
@@ -20,44 +20,87 @@
 
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
-@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.TimeUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReturnSummary
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
 
 @this(
         mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
         pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
         backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
 )
 
-@(isATrust: Boolean, backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
+@(previousReturn: Option[ReturnSummary], backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
```

Comment:
couldn't `previousReturn` be non-optional. In theory, you should always be able to have a reference to the `ReturnSummary` where the same completion date exists